pkg/planner/core/plan_to_pb.go

```diff
@@ -500,6 +500,31 @@ func (p *PhysicalIndexScan) ToPB(_ *base.BuildPBContext, _ kv.StoreType) (*tipb.
 	if p.NeedCommonHandle {
 		pkColIDs = tables.TryGetCommonPkColumnIds(p.Table)
 	}
+	if store == kv.TiFlash {
+		executorID := p.ExplainID().String()
+		queryColumns := make([]*model.ColumnInfo, 0, len(p.prop.FullTextProp.QueryColumns))
```

Comment:
I'm not sure if this `QueryColumns` is after column prune or the original `QueryColumns` that including all columns in the related index?

Reply:
`QueryColumns` represents the fields in the "match(a,b,c)" in SQL.